### PR TITLE
Enable more rubocop checks

### PR DESCRIPTION
**Pull request by eitoball**
Sorry for lots of changes. Most changes were done by rubocop autocorrect feature, so these should not be typos or such. Some changes were done manually mostly in `app/controllers/application_controller.rb`.

Once this PR is merged, we can enforce coding styles based on https://github.com/bbatsov/ruby-style-guide.

### CODE REVIEW

File: app/mailers/notifications.rb [hunk 1]
```diff
@@ -1,48 +1,53 @@
...
+  default from: 'mailer@safecast.org'
+  default_url_options[:locale] = 'en-US'
 
   def import_approved(import)
     @import = import
     mail(
-      :to => import.user.email,
-      :subject => "Your Safecast import has been approved - #{import.filename}")
+      to: import.user.email,
+      subject: 'Your Safecast import has been approved - ' << import.filename
```
Comment (matschaffer) [inline app/mailers/notifications.rb]: Looks like a bad rebase here

Comment (eitoball) [inline app/mailers/notifications.rb]: Fixed. Thanks for catching!

File: app/mailers/notifications.rb [hunk 2]
```diff
@@ -1,48 +1,53 @@
...
     @body = body
     @sender = sender
     mail(
-      :from => sender,
-      :to => import.user.email,
-      :subject => "Email from Safecast Moderator regarding your Safecast Import - #{import.filename}",
-      :body => body)
+      from: sender,
+      to: import.user.email,
+      subject: 'Email from Safecast Moderator regarding your Safecast Import - ' << import.filename,
```
Comment (matschaffer) [inline app/mailers/notifications.rb]: One more bad rebase

File: app/mailers/notifications.rb [hunk 3]
```diff
@@ -1,48 +1,53 @@
...
     @body = body
     @sender = sender
     mail(
-      :from => sender,
-      :to => import.rejected_by,
-      :subject => "Email from Safecast User regarding Safecast Import - #{import.filename}",
-      :body => body)
+      from: sender,
+      to: import.rejected_by,
+      subject: 'Email from Safecast User regarding Safecast Import - ' << import.filename,
```
Comment (matschaffer) [inline app/mailers/notifications.rb]: Make that two

Comment (eitoball) [inline app/mailers/notifications.rb]: Oh... I see. You fixed all. I thought 1 line... I guess that there are 3 more or something, then.

Comment (matschaffer) [inline app/mailers/notifications.rb]: Yep. Once I saw the other `<<`s figured may as well get them all more ruby-ish

File: app/mailers/notifications.rb (hunk 2 above)
Comment (eitoball) [inline app/mailers/notifications.rb]: Fixed.

File: app/mailers/notifications.rb (hunk 3 above)
Comment (eitoball) [inline app/mailers/notifications.rb]: Fixed. Thanks again.

**Pull request merged**